crowd-ui/src/App.test.tsx

```diff
@@ -3,14 +3,17 @@ import { render } from '@testing-library/react';
 import { Provider } from 'react-redux';
 import configureAppStore from './app/store';
 import App from './App';
+
 const { store } = configureAppStore();
 
+// eslint-disable-next-line no-undef
```

Comment:
instead of eslint ignore, you can add jest to your plugins

Reply:
because this is a test file and probably change later